README.md

```diff
@@ -55,10 +55,16 @@ val myModule = project
 
 Beside the baked-in SBT plugin, smithy4s comes with a CLI, that allows to generate Scala code and OpenAPI specs from smithy specs.
 
-We recommend using [coursier](https://get-coursier.io/docs/cli-launch) to run it
+We recommend using [coursier](https://get-coursier.io/docs/cli-launch) to install/run it
+
+#### Installation
+
+cs install --channel https://disneystreaming.github.io/coursier.json smithy4s
```

Comment:
I'd consider registering in https://github.com/coursier/apps/tree/master/apps-contrib/resources so that this becomes `cs install --contrib smithy4s`